docs/docs/api/turn-by-turn/api-reference.md

```diff
@@ -620,4 +620,5 @@ The codes correspond to code returned from a particular [Valhalla project](https
 |**5xx** | **Tyr project codes** |
 |500 | Failed to parse intermediate request format |
 |501 | Failed to parse TripDirections |
+|504 | GeoTIFF serialization not supported by service |
```

Comment:
you're _very_ diligent haha not sure this section has seen any updates since I joined the project